pkg/services/featuremgmt/registry.go

```diff
@@ -0,0 +1,125 @@
+package featuremgmt
+
+var (
+	// Register each toggle here
+	standardFeatureFlags = []FeatureFlag{
+		{
+			Name:            "caching",
+			Description:     "Temporarily store data source query results.",
+			State:           FeatureStateStable,
+			DocsURL:         "https://grafana.com/docs/grafana/latest/enterprise/query-caching/",
+			RequiresLicense: true,
+		},
```

Comment:
```suggestion
		{
			Name:            "caching",
			Description:     "Temporarily store data source query results.",
			State:           FeatureStateStable,
			DocsURL:         "https://grafana.com/docs/grafana/latest/enterprise/query-caching/",
			RequiresLicense: true,
		},
```
This feature toggle has been removed in https://github.com/grafana/grafana-enterprise/pull/1270